src/lib/components/LicenseField.js

```diff
@@ -0,0 +1,58 @@
+// This file is part of React-Invenio-Deposit
+// Copyright (C) 2020 CERN.
+// Copyright (C) 2020 Northwestern University.
+// Copyright (C) 2020 Cottage Labs LLP.
+//
+// React-Invenio-Deposit is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import React, { Component, useMemo } from 'react';
```

Comment:
can remove `useMemo` and other unused imports

Reply:
Thank you for the early review, and yep!